Packs/Lastline/Integrations/Lastline_v2/Lastline_v2_test.py

```diff
@@ -69,3 +70,77 @@ def test_get_report_context(path):
             json_obj = json_obj.read()
             out_dict = json.loads(json_obj)
         get_report_context(out_dict)
+
+
+def test_credentials_not_part_of_params(mocker):
+    """
+        When:
+            - testing connect to integration, using API key and API token only.
+
+        Then:
+            - validating that main finishes successfully without raising an error
+
+        """
+    mocker.patch.object(demisto, 'params', return_value={'url': 'testurl.com',
+                                                         'api_key': 'apikey',
+                                                         'api_token': 'apitoken'})
+    mocker.patch.object(demisto, 'command', return_value='test-module')
+    mocker.patch.object(Client, 'get_task_list', return_value=('Human readable', {}, {}))
+    assert main() is None
+
+
+def test_connect_with_credentials(mocker):
+    """
+    Given:
+        - connect cedentials
+
+    When:
+        - testing connect to integration, using credentials: email and password.
+
+    Then:
+        - validating no error is raised.
+
+    """
+    mocker.patch.object(demisto, 'params', return_value={'url': 'testurl.com',
+                                                         'credentials': {'identifier': 'identifier',
+                                                                         'password': 'password'}})
+    mocker.patch.object(demisto, 'command', return_value='test-module')
+    mocker.patch.object(Client, 'get_task_list', return_value=('Human readable', {}, {}))
+    client = Client(base_url='test.com', api_params={}, credentials={'username': 'identifier',
+                                                                     'password': 'password'})
+    assert client.test_module_command() == ('ok', {}, {})
+
+
+def raise_exception():
+    raise DemistoException('Authentication Error.')
+
+
+@pytest.mark.parametrize('params', [{'url': 'testurl.com',
+                                     'credentials': {'identifier': 'identifier',
+                                                     'password': 'password'}},
+                                    {'url': 'testurl.com',
+                                     'username': 'identifier',
+                                     'password': 'password-wrong'},
```

Comment:
What is this there no such option in the integration for param `username`. Did you mean to put here token and key?

Reply:
dont understand what you mean, but there is an error at these lines, fixed